plugins/woocommerce/src/Blocks/Shipping/ShippingController.php

```diff
@@ -488,4 +492,39 @@ function( $location ) {
 
 		return $served;
 	}
+
+	/**
+	 * Check if legacy local pickup is activated in any of the shipping zones or in the Rest of the World zone.
+	 *
+	 * @return bool
+	 */
+	public static function is_legacy_local_pickup_active() {
```

Comment:
This method basically does the same thing as the method added in `WC_Shipping`, would it make sense to avoid code duplication and just declare one method that can be used across the board? Or is there a specific reason to have two methods doing essentially doing the same thing?

Reply:
+1

Reply:
Ouch ... the duplication is a WIP leftover and, of course, having two methods that are doing the same, does not make sense. I'll remove the one in `ShippingController` and keep the one in `WC_Shipping`.